investor_wallet_platform_base/models/mail_template.py

```diff
@@ -34,13 +34,13 @@ def _get_email_template_dict(self):
                                readonly=True)
     iwp = fields.Boolean(string="IWP mail template")
 
-    def get_email_template_by_key(self, mail_template_key, structure):
+    def get_email_template_by_key(self, mail_template_key, structure, raise_=True):
```

Comment:
why underscore at the end of raise ?

Reply:
because it's a word of Python Language? :-)

Reply:
ok you mean it's a reserved word raise_error is more meaningful then... But I can live with it.